prometheus/cli.py

```diff
@@ -71,3 +73,36 @@ def metrics(host, token, interval, time, skip_namespaces, output, sort_by):
         print(yaml.dump(json.loads(std.read()), sort_keys=False))
     else:
         df.to_csv(sys.stdout)
+
+
+@main.command(help="Deploy process-exporter resources")
+def deploy():
+    oc_handler(oc.apply, "deployed.")
+
+
+@main.command(help="Delete process-exporter resources")
+def delete():
+    oc_handler(oc.delete, "deleted.")
+
+
+def oc_handler(func, msg):
+    files_path, list_of_files = get_data_file_dir()
+    for file in list_of_files:
+        with open(os.path.join(files_path, file), 'r') as resource:
+            try:
+                func(yaml.load(resource, Loader=yaml.FullLoader))
+                print(file.split('.')[0], msg)
+            except oc.model.OpenShiftPythonException as e:
+                print(e.msg, file.split('.')[0])
+
+
+def get_data_file_dir():
+    files_path = os.path.join(sys.exec_prefix, 'local', '.prom')
+    if not os.path.isdir(files_path):
+        files_path = os.path.join(sys.exec_prefix, '.prom')
+        if not os.path.isdir(files_path):
+            raise Exception("[ERROR] Couldn't find data files")
+    list_of_files = ["prometheusRules.yaml", "configmap.yaml",
+                     "service.yaml", "servicemonitor.yaml", "daemonset.yaml"]
+
+    return files_path, list_of_files
```

Comment:
@lack IDK if that's the right place for it, want to know what do you think

Reply:
Can't we just glob for `*.yaml` and install all of those?  Or concatenate then into a single yaml?

Reply:
Yes we can, I believe this will be smarter